chainer_compiler/chainer_compiler.py

```diff
@@ -265,23 +293,35 @@ def compile(self, inputs):
 
         self.compiled = True
 
+        params = dict(self.mc.namedparams())
+        if self.translator == 'onnx_chainer':
+            params = {'param' + key.replace('/', '_'): value for key, value
+                      in params.items()}
```

Comment:
I think we can update `params` by iterating `namedlinks` here. Maybe something like

```
for link_name, link in self.mc.namedlinks().items():
  if not isinstance(link, BatchNormalization):
    continue
  link_name = 'param' + link_name.replace('/', '_')
  for avg_name in ['avg_mean', 'avg_var']:
    key = link_name + '_' + avg_name
    assert key not in params
    params[key'] = getattr(link, avg_name)
```

Reply:
This approach looks concise. I modified the large part of the code to this kind of approach.